uint/src/uint.rs

```diff
@@ -813,6 +813,16 @@ macro_rules! construct_uint {
 				arr
 			}
 		}
+
+		impl From<[u8; $n_words * 8]> for $name {
+			fn from(bytes: [u8; $n_words * 8]) -> Self {
```

Comment:
why not just use https://github.com/paritytech/parity-common/blob/283b13f11440bc861b7374ebbd6fcccc3b8ecf74/uint/src/uint.rs#L777

?

Reply:
Yes, I'm reverting this to just to `bytes[..].as_ref().into()` (which will resolve to using `from_big_endian`).